### PR TITLE
WMS-472 | Update icons

### PR DESCRIPTION
**Pull request by jouderianjr**
#### :thinking: What?
Adding the new icons set provided By Sufiyan and fixing some spacing/colors/sizing regarding those icons.

#### :man_shrugging: Why?
Sufiyan added a bunch of new icons and renamed those as well and we need them for our awesome and fancy wms-web

#### :pushpin: Jira Issue
[WMS-472](https://paacklogistics.atlassian.net/browse/WMS-472)

#### :no_good: Blocked by
Blocked by a new release from [paack-ui-asset](https://github.com/PaackEng/paack-ui-assets/pull/1)

### :fire: Extra
Nothing probably 🤔 .

### CODE REVIEW

File: showcase/src/Navigators/Stories.elm [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            { title = "Vrau" 
+            , content = Nav.contentSingle Element.none
+            , dialog = Nothing
+            , hasMenu = False
+            }
+        view { navigatorsStories } =
+            Nav.navigator ForNav
+                navState
+                (toNavContainer appConfig context >> Nav.containerMap ForPage)
+                --|> Nav.withMenuPages (menuPageList appConfig.locale page)
```
Comment (miguelcoba) [inline showcase/src/Navigators/Stories.elm:49]: maybe remove this comment

**Review by miguelcoba — APPROVED** [review]

File: showcase/src/Navigators/Stories.elm (hunk 1 above)
Comment (jouderianjr) [reply showcase/src/Navigators/Stories.elm:49]: Oh boy, I totally forgot that I haven't finished that page :o hahaha, that's my bad, this is a test I was doing.

File: showcase/src/Navigators/Stories.elm [hunk 2]
```diff
@@ -0,0 +1,88 @@
+module Navigators.Stories exposing (stories, update)
```
Comment (PedroHLC) [inline showcase/src/Navigators/Stories.elm:1]: :scream_cat:  wow you added this new one

File: showcase/src/Model.elm [hunk 3]
```diff
@@ -19,17 +21,27 @@ type alias Model =
...
     , tabsStories : Tabs.Model
+    , navigatorsStories : Navigators.Model 
     , tabs : TabsPlugin.Model
     }
 
 
 init : Model
 init =
+    let
+        renderConfig =
```
Comment (PedroHLC) [inline showcase/src/Model.elm:32]: It's better to receive this guy from Main.
(Would be even better if we were receiving from UIExplorer/subscriptions)

File: showcase/src/Navigators/Stories.elm [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+            let
+                ( newState, subCmd ) =
+                    Nav.stateUpdate subMsg model.navState
+            in
+            ( { model | state = newState }
+            , Cmd.map Navigators.ForNav subCmd )
+
+
+stories renderConfig =
```
Comment (PedroHLC) [inline showcase/src/Navigators/Stories.elm:30]: :open_mouth:  you didn't rebased

File: showcase/src/Navigators/Stories.elm [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    storiesOf
+        "Checkboxes"
+        [ navigatorDemo renderConfig ]
+
+
+boxDemo renderConfig =
+    let
```
Comment (PedroHLC) [inline showcase/src/Navigators/Stories.elm:37]: move everything outside the let...

**Comment by jouderianjr** [comment]
@miguelcoba @PedroHLC I have decided to remove the Navigator story from this PR as is not exactly related and I still would need to do more work, I moved the changes to another branch.

**Pull request merged**